ci-operator/step-registry/ipi/conf/aws/oidc-creds/ipi-conf-aws-oidc-creds-commands.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+PREFIX="${SHARED_DIR}/manifest"
+cat >> "${PREFIX}_cluster-authentication-02-config.yaml" << EOF
+apiVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+  name: cluster
+spec:
+  serviceAccountIssuer: https://<something>-oidc-discovery.s3.amazonaws.com
+
+EOF
+
+CONFIG="[default]
+role_arn = arn:aws:iam::<something>:role/<something>-cred-minter-s3-role
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+"
+BASE64CONFIG=$(echo ${CONFIG} | base64)
+cat >> "${PREFIX}_secret-credentials-cred-minter-s3.yaml" << EOF
+apiVersion: v1
+data:
+  credentials: ${BASE64CONFIG}
+kind: Secret
+metadata:
+  name: cloud-credential-operator-s3-creds
+  namespace: openshift-cloud-credential-operator
+
+EOF
+
+CONFIG="[default]
+role_arn = arn:aws:iam::<something>:role/<something>-ebs-csi-driver-role
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+"
+BASE64CONFIG=$(echo ${CONFIG} | base64)
+cat >> "${PREFIX}_secret-credentials-ebs-csi-driver.yaml" << EOF
+apiVersion: v1
+data:
+  credentials: ${BASE64CONFIG}
+kind: Secret
+metadata:
+  name: cloud-credential-operator-s3-creds
+  namespace: openshift-cloud-credential-operator
+
+EOF
```

Comment:
this looks like a dupe of the same credentials right below this (and this one puts the Secret in the wrong namespace/name)